test/test_selectors.py

```diff
@@ -613,6 +536,116 @@ def test_leaking_fds(self):
         after_fds = len(proc.open_files())
         self.assertEqual(before_fds, after_fds)
 
+    def test_wait_io_close_is_called(self):
+        selector = self.SELECTOR()
+        self.addCleanup(selector.close)
+
+        def fake_constructor(*args, **kwargs):
+            return selector
+
+        old_selector = wait.DefaultSelector
+        wait.DefaultSelector = fake_constructor
+        self.addCleanup(setattr, wait, "DefaultSelector", old_selector)
+
+        rd, wr = self.make_socketpair()
+        wait.wait_for_write([rd, wr], 0.001)
+        self.assertIs(selector._map, None)
+
+    def test_wait_selector_error(self):
+        err = selectors.SelectorError(1)
+        self.assertEqual(err.__repr__(), "<SelectorError errno=1>")
+        self.assertEqual(err.__str__(), "<SelectorError errno=1>")
```

Comment:
This test isn't obviously anything to do with `wait`, so we should probably remove it from the name of the test.